tests/misc/tstreamsreadline.nim

```diff
@@ -0,0 +1,27 @@
+discard """
```

Comment:
the `discard` section here can be removed IMO since it doesn't test anything of interest; eg like `misc/tstrdist.nim`